htdocs/install/mysql/tables/llx_product.sql

```diff
@@ -51,14 +51,15 @@ create table llx_product
   recuperableonly               integer NOT NULL DEFAULT '0',       -- French NPR VAT
   localtax1_tx                  double(6,3)  DEFAULT 0,
   localtax1_type                varchar(10)  NOT NULL DEFAULT '0',
-  localtax2_tx                  double(6,3)  DEFAULT 0, 
+  localtax2_tx                  double(6,3)  DEFAULT 0,
   localtax2_type                varchar(10)  NOT NULL DEFAULT '0',
   fk_user_author                integer DEFAULT NULL,               -- user making creation
   fk_user_modif                 integer,                            -- user making last change
   tosell                        tinyint      DEFAULT 1,             -- Product you sell
   tobuy                         tinyint      DEFAULT 1,             -- Product you buy
   onportal                      tinyint      DEFAULT 0,	            -- If it is a product you sell and you want to sell it on portal (module website must be on)
   tobatch                       tinyint      DEFAULT 0 NOT NULL,    -- Is it a product that need a batch management (eat-by or lot management)
+  batch mask			        varchar(32),                        -- If the product has batch feature, you may want to use a batch mask per product
```

Comment:
No spaces are allowed into a field name. Can you use
batch_mask instead of batch mask

Reply:
of course, it's a bad typing mismatch !

Reply:
@eldy : fixed with https://github.com/Dolibarr/dolibarr/pull/16996/commits/0adbffd8645457abc155169f9461f8a0b0cd36d1